### PR TITLE
Add support for dark mode

### PR DESCRIPTION
**Pull request by Vlada31R**

### CODE REVIEW

**Comment by fishcharlie** [comment]
Any updates on this? It looks like there are build errors from CircleCI.

**Comment by Vlada31R** [comment]
CircleCI builds using Xcode 10.2.0, that's why iOS 13 colors could't be supported

**Comment by fishcharlie** [comment]
@Vlada31R CircleCI supports Xcode 11. https://circleci.com/docs/2.0/testing-ios/#supported-xcode-versions

So it's something that can be fixed by updating the CircleCI config file.

**Review by alexmomotiuk — APPROVED** [review]

**Review by hyouuu — APPROVED** [review]
This is great!

**Comment by hyouuu** [comment]
I updated the circle config but don't have access to circle to re-run it - could you pull then push again?

**Comment by SD10** [comment]
Thank you for contributing to MessageKit! I've invited you to join the MessageKit GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/MessageKit/MessageKit/blob/master/CONTRIBUTING.md) and join the [MessageKit Slack channel](https://join.slack.com/t/messagekit/shared_invite/enQtMjI4NzIzNzMyMzU0LTEyNmYzNzBlNjNiYjAwYjY1ZjhkMmVjNDE4OGNlMDY1OTRmNzVmMzI4MTg3OWQzZTM2NGU4NTU3ZDE5NzkwNzQ). Feel free to reach out if you have any questions! :smiley:

**Comment by hyouuu** [comment]
It should pass after config update, so just merged

File: Sources/Extensions/UIColor.swift [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+//  UIColor.swift
+//  MessageKit
+//
+//  Created by Vlada Radchenko on 9/30/19.
+//  Copyright © 2019 MessageKit. All rights reserved.
+//
+
+import Foundation
+
+extension UIColor {
```
Comment (austinwright) [inline Sources/Extensions/UIColor.swift:11]: Any reason these are in a separate file rather than adding them to UIColor+Extensions.swift ?

Comment (hyouuu) [reply Sources/Extensions/UIColor.swift:11]: He probably didn't see it. Can make another PR to move these when I get free time, or it's even better if @Vlada31R could do it

**Pull request merged**